### PR TITLE
Update technical-stack.md

### PR DESCRIPTION
**Pull request by varmoh**
Added descriptions for TIM
Added a "Other" paragraph for services that were not categorized.

### CODE REVIEW

File: documents/technical-stack.md [hunk 1]
```diff
@@ -19,3 +19,7 @@ _TO DO_
 
 ## Turvis
 _TO DO_
+
+## Other
```
Comment (turnerrainer) [inline documents/technical-stack.md]:
```suggestion
```

File: documents/technical-stack.md [hunk 2]
```diff
@@ -19,3 +19,7 @@ _TO DO_
 
 ## Turvis
 _TO DO_
+
+## Other  
+Customer-service, is backend tool for administrating purposes to be used by customer service agent and admins
```
Comment (turnerrainer) [inline documents/technical-stack.md]:
```suggestion
## Back-office GUI
A GUI-based tool for administrating purposes to be used by CSA and administrators
```

File: documents/technical-stack.md [hunk 3]
```diff
@@ -19,3 +19,7 @@ _TO DO_
 
 ## Turvis
 _TO DO_
+
+## Other  
+Customer-service, is backend tool for administrating purposes to be used by customer service agent and admins  
+Chat-widget, is frontend tool to be used for interaction between user and bot/customer service agent
```
Comment (turnerrainer) [inline documents/technical-stack.md]:
```suggestion
## Chat-widget
A frontend tool to be used for interaction between End User and Chatbot / CSA
```

**Review by turnerrainer — CHANGES_REQUESTED** [review]
Some updates

**Review by varmoh — COMMENTED** [review]
Changes were made to word and explain descriptions better

**Comment by varmoh** [comment]
Descriptions were changed

**Pull request merged**